pkg/servicecentral/client_test.go

```diff
@@ -393,34 +394,6 @@ func TestGetServiceWithFailedAttributesCall(t *testing.T) {
 	require.Equal(t, 2, handler.RequestSnapshots.Calls())
 }
 
-func TestGetServiceWithMultipleOpsGenieAttribute(t *testing.T) {
```

Comment:
This test does not make sense anymore as it was making sure we errored on multiple OG attributes, this is no longer an error case as we handle multiple attributes later on.